src/java.xml/share/conf/jaxp-compat.properties

```diff
@@ -3,9 +3,27 @@
 #
 # ** Warning: unsafe configuration, use with security assessment **
 #
-# jaxp-compat.properties is a JAXP configuration file with compatible settings
-# prior to a strict configuration such as jaxp-strict.properties. In this
-# configuration, properties that have more restrictive settings as in the
+# This is one of the three configuration files provided in the JDK:
+#
+# jaxp.properties: this is the default configuration that the JDK uses to set
+# property values when XML factories are initiated.
+#
+# jaxp-strict.properties: this file resembles what will become the Secure-By-Default
+# configuration where a strict restriction is the default. This file allows
+# deployments to test the more secure/strict behavior, identify issues such as
+# a processor unknowingly makes outbound network connections to fetch DTD,
+# or processes XML that relies on extension functions
+#
+# jaxp-compat.properties: this file can be used to regain compatibility once the
+# JDK has switched to a strict configuration as indicated in jaxp-strict.properties.
+# This configuration contains the same properties as those in jaxp-strict.properties
+# except it sets them back to the current status of the JDK. Note that, although
```

Comment:
'....sets them back to the current status of the JDK'

I think you are trying to indicate that this property file specifies the JAXP property values that were in place prior to being More Secure?

Reply:
Thanks Lance for the offline discussion. Updated the description.